Frameworks/Foundation/NSSet.mm

```diff
@@ -370,6 +370,9 @@ + (NSSet*)set {
  @Status Interoperable
 */
 - (BOOL)containsObject:(id)obj {
+    if (obj == nil) {
+        return NO;
+    }
```

Comment:
please move this to member method instead. We are getting a new version of this class and that will make merging this easier.